package.json

```diff
@@ -100,9 +100,6 @@
 		"xdg-basedir": "^5.1.0",
 		"zod": "^3.25.67"
 	},
-	"overrides": {
-		"vite": "npm:rolldown-vite@latest"
-	},
```

Comment:
Disabling this in the main package as well for consistency, but we can consider keeping it